disk/development/ontology.ttl

```diff
@@ -388,6 +463,15 @@ xsd:date rdf:type rdfs:Datatype .
                  rdfs:comment "Contains a workflow and a variable binding"@en ;
                  rdfs:label "Workflow binding"@en .
 
+###  http://disk-project.org/ontology/disk#Question
+:Question rdf:type owl:Class ;
+                 rdfs:comment "Contains a hypothesis question"@en ;
```

Comment:
You are not defining question here. You are using the same term in the definition